mempool/mempool.go

```diff
@@ -147,6 +147,8 @@ type Mempool struct {
 	preCheck             PreCheckFunc
 	postCheck            PostCheckFunc
 
+	sizeBytes int64 // size of all txs in the mempool in bytes
```

Comment:
Is it on purpose here `int64` and in Mempool config `int`? The same regarding comment as above.